vesta/tables/clash_of_code.py

```diff
@@ -0,0 +1,28 @@
+from typing import Optional
+
+import sqlalchemy as db
+
+from . import Base
+from .. import session_maker
+from ..services import clash_of_code_helper, ClashOfCodeGame
+
+session = session_maker()
+
+class ClashOfCodeGuildGame(Base):
```

Comment:
Why is it not just "ClashOfCodeGame" if there is only one per guild?

Reply:
As said maybe use a foreign key from guild to link it up

Reply:
ClashOfCodeGame is already used to talk about the representation of an api object